src/index.js

```diff
@@ -14,11 +15,19 @@ function populate() {
     box.type = 'checkbox';
     box.className = 'checkbox';
     box.id = `checkbox${toDoList.arrayTasks[i].index}`;
+    box.checked = toDoList.arrayTasks[i].completed;
     div.appendChild(box);
     const task = document.createElement('input');
     task.className = 'taskToDo';
     task.id = `taskToDo${toDoList.arrayTasks[i].index}`;
     task.value = toDoList.arrayTasks[i].description;
+    if (box.checked) {
+      task.classList.add('taskToDoChecked');
+      task.style.color = 'rgb(175,175,175)';
+    } else {
+      task.classList.remove('taskToDoChecked');
+      task.style.color = 'inherit';
+    }
```

Comment:
- [ ] Its very nice adding indexing to your todo list, please make sure that when we click on the checklist it should be `completed: true` right now when we click it is still `completed: false`. Kindly fix that.
#### Your todo👇
![todo](https://user-images.githubusercontent.com/66992901/200381773-341e1543-f09c-47db-8df6-d3d34272ff67.PNG)

Reply:
I checked it and worked fine. I even cloned it to another folder and the HTML file in the dist folder worked as expected in several folders.

Reply:
Its good that it works properly but in my machine I cloned it and tried many ways but still didn't work.